src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  position: relative;
+
+  font-size: 12px;
+  font-weight: $medium-font;
+  text-transform: uppercase;
+  white-space: nowrap;
+  text-decoration: none;
+  text-align: center;
+  color: $primary-text-color;
+}
+
+.nav__link:hover {
+  color: $main-color;
+}
+
+.is-active {
+  color: $main-color;
+}
+
+.is-active::after {
+  content: '';
+  height: 4px;
+  background-color: $main-color;
+  border-radius: 8px;
+  width: 100%;
+
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
```

Comment:
Similarly, the `.is-active::after` pseudo-element should be nested within the `.is-active` class definition to keep the SCSS organized and maintain the relationship between the selector and its pseudo-element.